src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -17,6 +17,11 @@ export class VeteranBenefitSummaryLetter extends React.Component {
   }
 
   handleChange(domEvent) {
+    window.dataLayer.push({
+      event: 'letter-benefit-option-clicked',
+      'letter-benefit-option': domEvent.target.id,
```

Comment:
Yeah, I agree this looks cleaner than packing all the identifiers into the event name.

I was just browsing the GTM docs and don't understand their comment in this section: https://developers.google.com/tag-manager/devguide#multipush which says that pushing a variable with the same name and a new value will overwrite the old value (e.g., for `letter-benefit-option`, which will take on various values as the user clicks on different checkboxes). Their comment doesn't make sense since `dataLayer` is an array, so pushing should just add a new one. Maybe just poorly written documentation, and you will indeed have access to all of the pushed variables on the reporting side?